fe/fe-core/src/main/java/com/starrocks/catalog/TabletStatMgr.java

```diff
@@ -132,7 +132,11 @@ private void updateTabletStat(Long beId, TTabletStatResult result) {
                 continue;
             }
             // TODO(cmy) no db lock protected. I think it is ok even we get wrong row num
-            replica.updateStat(entry.getValue().getData_size(), entry.getValue().getRow_num());
+            replica.updateStat(
+                    entry.getValue().getData_size(),
+                    entry.getValue().getRow_num(),
+                    entry.getValue().getVersion_count()
```

Comment:
Will it work for BE of older versions?

Reply:
I've tested it and there's nothing wrong with it

Reply:
ok then.